### PR TITLE
[11.0][MIG] web_send_message_popup: Migrate to 11.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: web_send_message_popup/__manifest__.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+    'version': '11.0.1.0.0',
+    'author': "Camptocamp, Odoo Community Association (OCA)",
+    'maintainer': 'Camptocamp',
+    'license': 'AGPL-3',
+    'category': 'Hidden',
+    'depends': [
+        'web',
+        'mail',
+    ],
+    'website': 'http://www.github.com/OCA/web',
```
Comment (grindtildeath) [inline web_send_message_popup/__manifest__.py]: https would have been better :wink:

**Review by grindtildeath — APPROVED** [review]

File: web_send_message_popup/__manifest__.py (hunk 1 above)
Comment (simahawk) [reply web_send_message_popup/__manifest__.py]: and no "www" :)

File: web_send_message_popup/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'category': 'Hidden',
+    'depends': [
+        'web',
+        'mail',
+    ],
+    'website': 'https://www.github.com/OCA/web',
+    'data': [
+        'templates/assets.xml',
+    ],
+    'auto_install': False,
```
Comment (simahawk) [inline web_send_message_popup/__manifest__.py]: remove this

**Review by simahawk — CHANGES_REQUESTED** [review]
tested functionally: works even if bit flickering as it renders 1st the std widget and then shows the other one. I've checked the code and unfortunately seems there's no good way to hook if not overriding whole big functions.
Please, check comments on manifest.

**Review by simahawk — APPROVED** [review]
@arkostyuk LG. Would be nice to keep [my comment](https://github.com/OCA/web/pull/1051#pullrequestreview-161987165) on flickering behavior in roadmap :wink:

**Comment by ghost** [comment]
@simahawk roadmap ~updated~ added, actually :slightly_smiling_face:

**Review by tschanzt — APPROVED** [review]
looks good.

**Pull request merged**